monitoring-micrometrics-grafana-prometheus/src/main/java/org/apache/camel/example/spring/boot/monitoring/Application.java

```diff
@@ -93,4 +99,29 @@ public void afterApplicationStart(CamelContext camelContext) {
             }
         };
     }
-}
\ No newline at end of file
+
+    @Bean("dummyapisSslContext")
```

Comment:
Please add a comment why this is needed

Reply:
do you think clear enough? alternatively we can change the api to invoke, but I would not to change the behavior of the example